server/database/dbSeed.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+const mocker = require('mocker-data-generator').default;
+const db = require('./index');
+
+// mock data template for review data
+const reviewGenerator = {
+  url_id: {
+    faker: 'random.number({"min": 0, "max": 99})',
+  },
+  name: {
+    faker: 'name.firstName',
+  },
+  location: {
+    faker: 'address.state',
+  },
+  date: {
+    faker: 'date.past',
+  },
+  comment: {
+    faker: 'lorem.paragraph',
+  },
+  raiting: {
+    faker: 'random.number({"min": 1, "max": 5})',
+  },
+};
+
+// generates a random number of data enteries(0 - 100) a random number of times ((5 - 35))
+const seeder = () => {
+  let counter = (Math.floor(Math.random() * 35) + 5);
+  const promises = [];
+  while (counter > 0) {
+    promises.push(mocker()
+      .schema('reviewGenerator', reviewGenerator, Math.floor(Math.random() * 100))
+      .build()
+      .then((info) => {
+        info.reviewGenerator.forEach((review) => {
+          const singleReviewArguments = [review.url_id, review.name, review.location, review.date,
+            review.comment, review.raiting];
```

Comment:
@Matt-Wigg, I believe you could improve the code readability by isolating this piece in a separate function that, when called, invokes the db insertion function, and returns a promise to be added to the promises array.